adapters/InMobi/InMobiAdapter/GADMAdapterInMobiBannerAd.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+#import <InMobiSDK/IMBanner.h>
+
+@interface GADMAdapterInMobiBannerAd : NSObject<GADMediationBannerAd, IMBannerDelegate>
```

Comment:
Can you rename this to: `GADMAdapterInMobiBannerAdLoader`?

Reply:
Also we can create an `AdLoader` directory and put all the banner, interstitial, rewarded and native ad header and implementation files in it.